lib/node.js

```diff
@@ -1720,6 +1720,8 @@ declare module "util" {
   declare function inherits(constructor: Function, superConstructor: Function): void;
   declare function deprecate(f: Function, string: string): Function;
   declare function promisify(f: Function): Function;
+  declare export var TextEncoder: Class<TextEncoder>;
+  declare export var TextDecoder: Class<TextDecoder>;
```

Comment:
If I don't use the `export` (like for others module), flow doesn't find `TextEncoder`... but finds `TextDecoder`, for my test file. Super strange.